src/App.jsx

```diff
@@ -25,21 +27,23 @@
 
 export const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">Maksym is my husband</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+    <Person
+      person={{
+        name: 'Misha',
+        age: 37,
+        sex: 'm',
+        isMarried: true,
+        partnerName: 'Natasha',
+      }}
+    />
+    <Person
+      person={{
+        name: 'Olya',
+        sex: 'f',
+        isMarried: true,
+        partnerName: 'Maksym',
+      }}
+    />
```

Comment:
Similarly, pass the entire `olya` object to the `Person` component as a prop, like `<Person person={olya} />`, instead of individual fields.